pints/_error_measures.py

```diff
@@ -272,6 +272,35 @@ def __call__(self, x):
             (self._problem.evaluate(x) - self._values)**2))
 
 
+class NormalisedRootMeanSquaredError(ProblemErrorMeasure):
+    """
+    Calculates a normalised root mean squared error (RMSE):
+    ``f = N * sqrt( sum( (x[i] - y[i])**2 / n) )``
+    where ``N`` is a normalisation constant defined as the inverse of the RMSE
+    of the data ``y`` with respect to zero.
```

Comment:
Is the error-w.r.t.-zero way the best way to describe it? Is it conceptually an error w.r.t. a zero signal? Or do you derive the measure by thinking of sqrt(sum(y**2)) as some kind of magnitude?

Reply:
Do you have any suggestion on how to word it?

Reply:
Maybe just write out the equation in full so you don't need to use too many words?